frame/bounties/src/lib.rs

```diff
@@ -78,13 +78,15 @@ mod tests;
 mod benchmarking;
 pub mod weights;
 
-use sp_std::prelude::*;
+use sp_std::{
+	prelude::*,
+};
 
 use frame_support::{decl_module, decl_storage, decl_event, ensure, decl_error};
 
 use frame_support::traits::{
 	Currency, Get, Imbalance, OnUnbalanced, ExistenceRequirement::{AllowDeath},
-	ReservableCurrency};
+	ReservableCurrency, BalanceStatus as Status};
```

Comment:
```suggestion
	ReservableCurrency, BalanceStatus};
```

Not much point to rename it